outline.md

```diff
@@ -294,7 +292,16 @@
     - firewalls
 - communicating
   - phrasing and terminology
-  
+- logging
```

Comment:
this was a pure cut paste from the previous 2 days of troubleshooting and this can be cut down